entity-types/infra-awsmqexchange/definition.yml

```diff
@@ -0,0 +1,7 @@
+domain: INFRA
+type: AWSMQEXCHANGE
+configuration:
```

Comment:
I guess we would also need dashboards, summary and golden metrics / ui definitions in case we are introducing a new entity type ?
([docs](https://github.com/newrelic/entity-definitions/tree/main/docs/entities))